src/deploy/osp_deployer/checkpoints.py

```diff
@@ -288,9 +288,9 @@ def rhscon_vm_health_check(self):
 
         logger.debug("*** Verify the Storage Console VM can ping "
```

Comment:
Nothing urgent, but we might need to change the name here away from Storage Console

Reply:
NP.